sdk/examples/space/camera-acquisitor-system/src/main/java/esa/mo/nmf/apps/CameraAcquisitorSystemCameraHandler.java

```diff
@@ -307,7 +307,8 @@ public void takePhotograph(long actionInstanceObjId, int stageOffset, int totalS
         new Duration(casMCAdapter.getExposureTime()),
         casMCAdapter.getGainRed(),
         casMCAdapter.getGainGreen(),
-        casMCAdapter.getGainBlue());
+        casMCAdapter.getGainBlue(),
```

Comment:
The API is broken here -> can we patch the StubGenerator to generate multiple constructor wrappers for composites with optional fields?

Reply:
Yeah, I like the idea. Will come with MOv9